homework28/src/test/java/com/vadim/tkach/lesson28/ShapeTest.java

```diff
@@ -0,0 +1,64 @@
+package com.vadim.tkach.lesson28;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+class ShapeTest {
```

Comment:
I'd recommend to split the tests into `RectangleTest`, `SquareTest`, `CircleTest`